northstar/src/runtime/pipe.rs

```diff
@@ -13,7 +13,7 @@
 //   limitations under the License.
 
```

Comment:
the pipe module is kind of hard to read and would certainly benefit from some explaining comments